platforms/Windows/cli.wxs

```diff
@@ -0,0 +1,266 @@
+
+    <ComponentGroup Id="libclang">
+      <Component Directory="_usr_bin" Id="libclang.dll">
+        <File Id="libclang.dll" Source="$(var.TOOLCHAIN_ROOT)\usr\bin\libclang.dll" Checksum="yes" KeyPath="yes" />
+      </Component>
+
+      <Component Directory="_usr_lib" Id="libclang.lib">
+        <File Id="libclang.lib" Source="$(var.TOOLCHAIN_ROOT)\usr\lib\libclang.lib" Checksum="yes" KeyPath="yes" />
+      </Component>
+
+      <Component Directory="_usr_include_clang_c" Id="BuildSystem.h">
+        <File Id="BuildSystem.h" Source="$(var.TOOLCHAIN_ROOT)\usr\include\clang-c\BuildSystem.h" Checksum="yes" KeyPath="yes" />
+      </Component>
+      <Component Directory="_usr_include_clang_c" Id="CXCompilationDatabase.h">
+        <File Id="CXCompilationDatabase.h" Source="$(var.TOOLCHAIN_ROOT)\usr\include\clang-c\CXCompilationDatabase.h" Checksum="yes" KeyPath="yes" />
+      </Component>
+      <Component Directory="_usr_include_clang_c" Id="CXErrorCode.h">
+        <File Id="CXErrorCode.h" Source="$(var.TOOLCHAIN_ROOT)\usr\include\clang-c\CXErrorCode.h" Checksum="yes" KeyPath="yes" />
+      </Component>
+      <Component Directory="_usr_include_clang_c" Id="CXString.h">
+        <File Id="CXString.h" Source="$(var.TOOLCHAIN_ROOT)\usr\include\clang-c\CXString.h" Checksum="yes" KeyPath="yes" />
+      </Component>
+      <Component Directory="_usr_include_clang_c" Id="Documentation.h">
+        <File Id="Documentation.h" Source="$(var.TOOLCHAIN_ROOT)\usr\include\clang-c\Documentation.h" Checksum="yes" KeyPath="yes" />
+      </Component>
+      <Component Directory="_usr_include_clang_c" Id="FatalErrorHandler.h">
+        <File Id="FatalErrorHandler.h" Source="$(var.TOOLCHAIN_ROOT)\usr\include\clang-c\FatalErrorHandler.h" Checksum="yes" KeyPath="yes" />
+      </Component>
+      <Component Directory="_usr_include_clang_c" Id="Index.h">
+        <File Id="Index.h" Source="$(var.TOOLCHAIN_ROOT)\usr\include\clang-c\Index.h" Checksum="yes" KeyPath="yes" />
+      </Component>
+      <Component Directory="_usr_include_clang_c" Id="Platform.h">
+        <File Id="Platform.h" Source="$(var.TOOLCHAIN_ROOT)\usr\include\clang-c\Platform.h" Checksum="yes" KeyPath="yes" />
+      </Component>
+      <Component Directory="_usr_include_clang_c" Id="Refactor.h">
+        <File Id="Refactor.h" Source="$(var.TOOLCHAIN_ROOT)\usr\include\clang-c\Refactor.h" Checksum="yes" KeyPath="yes" />
+      </Component>
+    </ComponentGroup>
+
+    <ComponentGroup Id="IndexStore">
+      <Component Directory="_usr_bin" Id="libIndexStore.dll">
+        <File Id="libIndexStore.dll" Source="$(var.TOOLCHAIN_ROOT)\usr\bin\libIndexStore.dll" Checksum="yes" KeyPath="yes" />
+      </Component>
+
+      <Component Directory="_usr_lib" Id="libIndexStore.lib">
+        <File Id="libIndexStore.lib" Source="$(var.TOOLCHAIN_ROOT)\usr\lib\libIndexStore.lib" Checksum="yes" KeyPath="yes" />
+      </Component>
+
+      <Component Directory="_usr_include_indexstore" Id="indexstore.h">
```

Comment:
Yes, this is used by IndexStoreDB which is part of LSP.  This allows development against a toolchain for LSP without building the full compiler.

Reply:
I don't think that we want to package the debug symbols ever:
  1. it would make the installer download ~10-20G rather than ~400M (which is already large)
  2. it is not a common practice on Windows
  3. we really want to use a symbol server to auto-match the PDB for the binary

I can split this up into two MSIs, but then I think that we go down the slipper slope of what is the right place to break up the MSIs (i.e. perhaps BlocksRuntime and dispatch should be separate MSIs as well).